### ISSUE TITLE
warning that nothing works

### ISSUE THREAD

**Issue by osp54** [artbits/quickio#1]
30, 2022 6:25:46 �� com.caucho.hessian.io.SerializerFactory getDeserializer
WARNING: Hessian/Burlap: 'org.xcore.plugin.comp.Database$PlayerData' is an unknown class in jdk.internal.loader.ClassLoaders$AppClassLoader@2a139a55:

```java
public class PlayerData extends QuickIO.Object {
    public String uuid;
    public Integer wins = 0;

    public PlayerData(String uuid) {
        this.uuid = uuid;
    }

    public PlayerData(String uuid, Integer wins) {
        this.uuid = uuid;
        this.wins = wins;
    }
}
```
`var data = db.findFirst(PlayerData.class, p -> p.uuid.equals(uuid));`

**Comment by artbits** [comment]
### Cause:
After you use the PlayerData class to store data, have you modified the directory path of the PlayerData class? If there are modifications, this warning will appear.

QuickIO internally uses the Hessian library for data serialization, and Hessian records the package name of the Java bean. If you modify the directory path of Java beans, After byte[] is converted to object, the original Java bean cannot be found, which causes Hessian to record the field data in HashMap, and finally the data cannot be transferred to the new Java bean.
In short, the original data still exists, but cannot be extracted.

### Resolvent:
1. After creating a Java bean, try not to modify its directory.
2. Later, I will consider adding a data repairer to repair the Java bean data in the original directory.
